src/fu-history.c

```diff
@@ -325,6 +343,10 @@ fu_history_create_or_migrate (FuHistory *self, guint schema_ver, GError **error)
 		g_debug ("migrating v%u database by altering", schema_ver);
 		if (!fu_history_migrate_database_v4 (self, error))
 			return FALSE;
+	} else if (schema_ver == 5) {
+		g_debug ("migrating v%u database by altering", schema_ver);
+		if (!fu_history_migrate_database_v5 (self, error))
+			return FALSE;
```

Comment:
don't you also need to do this if someone goes from 1->6 and 2->6 and 3->6 etc?

Reply:
Yes, good catch, fixed.